examples/asr/asr_streaming/speech_to_text_streaming_infer.py

```diff
@@ -183,7 +205,31 @@ def main():
         action="store_true",
         help="Whether to compare the output of the model with the offline mode.",
     )
-    parser.add_argument("--batch_size", type=int, default=32)
+    parser.add_argument(
+        "--batch_size",
+        type=int,
+        default=32,
+        help="The batch size to be used to perform streaming in batch mode with multiple streams",
+    )
+    parser.add_argument(
+        "--chunk_size",
+        type=int,
+        default=-1,
+        help="The chunk_size to be used for models trained with full context and offline models",
+    )
+    parser.add_argument(
+        "--shift_size",
+        type=int,
+        default=-1,
+        help="The shift_size to be used for models trained with full context and offline models",
+    )
+    parser.add_argument(
+        "--left_chunks",
+        type=int,
```

Comment:
How about more explicit name - --offline_left_chunks

Reply:
These parameters are not specific to offline models. For models with limited context, these values have some default values but they can get controlled by the user even for streaming models.